src/ResourceManager/CognitiveServices/Commands.Management.CognitiveServices/Properties/AssemblyInfo.cs

```diff
@@ -46,5 +46,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 
-[assembly: AssemblyVersion("0.9.1")]
-[assembly: AssemblyFileVersion("0.9.1")]
+[assembly: AssemblyVersion("0.9.2")]
```

Comment:
@felixwa you can revert the change to these versions -- we will update the versions during the release